Tribler/Core/exceptions.py

```diff
@@ -48,6 +48,15 @@ def __init__(self, msg=None):
         TriblerException.__init__(self, msg)
 
 
+class DuplicateChannelNameError(TriblerException):
+
+    """ The Channel name already exists in the ChannelManager channel list,
+    i.e., one of your own Channels with the same name already exists. """
+
+    def __init__(self, msg=None):
+        TriblerException.__init__(self, msg)
```

Comment:
It is a direct copypasta from the existing TriblerExceptions. Do you want me to refactor all exception classes?